db/migrate/20230601161704_make_neighbour_letter_neighbour_unique.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class MakeNeighbourLetterNeighbourUnique < ActiveRecord::Migration[7.0]
+  def up
+    remove_index :neighbour_letters, :neighbour_id
+    change_column :neighbour_letters, :neighbour_id, :bigint, unique: true
+    add_index :neighbour_letters, :neighbour_id, unique: true, name: "ix_neighbour_letters_on_neighbour_id"
```

Comment:
Will this be an issue when we get round to being able to resend letters to the same neighbour?

Reply:
I was thinking in terms of a resent letter being the same NeighbourLetter, but perhaps that wouldn't actually make sense.

If we did it this way, then when resending we'd throw away the original Notify ID in favour of the new one, which might be OK but if we want to keep track of the fact that it was resent then perhaps it would be better to have a one-to-many relationship here (one neighbour can have multiple letters).

In which case, we could still check that a letter hasn't been sent by looking at the one-to-many, so that part would still work.

Reply:
Yeah I reckon it'll have to be one neighbour having multiple letters. In addition to resending, it's unlikely but possible that there are several planning applications surrounding that neighbour too which they'd get a separate letter for

Reply:
>it's unlikely but possible that there are several planning applications surrounding that neighbour too which they'd get a separate letter for

This wouldn't be an issue. A Neighbour is unique per Consultation, so the same address might be represented as more than one Neighbour if there are several Consultations close enough to each other, and their status would be tracked individually. I don't think there would be any need for us to know/care that two instances of 60 Commercial Street are actually the same.

Reply:
OK, I've removed this migration, but for now the logic about when to send doesn't account for this — we can fix it once we actually add the resending functionality.